modules/ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries.adoc

```diff
@@ -0,0 +1,384 @@
+:_content-type: PROCEDURE
+[id="ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries_{context}"]
+= Configuring the hub cluster to use unauthenticated registries
+
+Use the following procedure to configure the hub cluster to use unauthenticated registries.
+
+.Prerequisites
+
+. Install <https://github.com/openshift/hive/blob/master/docs/install.md>[Hive]
+. Install the <https://github.com/openshift/hive/blob/master/docs/using-hive.md#cluster-provisioning>[ClusterDeployment Custom Resource Definition (CRD)]
+. Install <https://sdk.operatorframework.io/docs/installation>[operator-sdk]
+. Install <https://github.com/kubernetes-sigs/kustomize/releases>[kustomize]
+. Install <https://github.com/operator-framework/operator-registry/releases>[opm]
+
+.Procedure
+
+. Build the operator bundle (optional)
```

Comment:
Sorry, where in the official docs will this be located? The reason I ask is because the instructions for this procedure below directly refers to the assisted-service github repo

Reply:
Hi, @CrystalChun. The content is going in Scalability and Performance after [Configuring the hub cluster to use a disconnected mirror registry](https://docs.openshift.com/container-platform/4.12/scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.html#ztp-configuring-the-cluster-for-a-disconnected-environment_ztp-preparing-the-hub-cluster) as part of 4.12.z. Thanks!